apps/i18n/music/en_us.json

```diff
@@ -91,6 +91,16 @@
   "packModePopular": "Popular",
   "packModeSong": "Song",
   "packModeArtist": "Artist",
+  "patternAiClickDrums": "Click to set up the start of your drums",
+  "patternAiClickDrums3": "Click to set up the start of your drums.\nThree more!",
+  "patternAiClickDrums2": "Click to set up the start of your drums.\nTwo to go!",
+  "patternAiClickDrums1": "Click to set up the start of your drums.\nOne more to go!",
```

Comment:
Should we just break out the "Three more!"/"Two to go!" strings and combine with `patternAiClickDrums` in code?

Reply:
I'm always slightly apprehensive about recombining strings, just in case there's a language in which the paragraph is better translated as a whole.